advanced/am-pattern-2/templates/am-analytics/worker/wso2am-pattern-2-am-analytics-worker-deployment.yaml

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@Chathuru license header year needs to be updated. Applicable to other locations as well.

Reply:
Fix in https://github.com/Chathuru/kubernetes-apim/commit/0205f6311ce5b9ff81ebe108aa79ff7a040078e1 commit